common/src/app/common/types/container.cljc

```diff
@@ -62,6 +63,17 @@
   [container shape-id f]
   (update-in container [:objects shape-id] f))
 
+(defn get-component-shape
```

Comment:
In file.cljs there is a helper with the same name that does another thing... Maybe it's better to use another name?

Reply:
It's possible. When we drop the components-v1 compatibility and remove container abstraction, I want to do a general review of helpers. Until then, I'm documenting all this and thinking of a glossary of terms, and will review many names.